liberty-archetype-ear/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>net.wasdev.wlp.maven</groupId>
+        <artifactId>liberty-maven</artifactId>
+        <version>2.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>liberty-archetype-ear</artifactId>
+    
+    <packaging>maven-archetype</packaging>
+    
+    <name>Archetype - liberty-archetype-ear</name>
+    
+    <description>
+        An archetype that generates a Java EE 7 multi-module project that includes a EJB module, a Web application module and a Ear module
```

Comment:
Same comment - 'a EJB module, a web application module and a EAR module' need to be changed to 'an EJB module, a web application module and an EAR module.'